JavaScript/JavaScript.sublime-syntax

```diff
@@ -983,8 +983,11 @@ contexts:
       fail: for-in-of
 
   for-in-of-word:
-    - match: (?:of|in){{identifier_break}}
-      scope: keyword.operator.word.js
+    - match: in{{identifier_break}}
+      scope: keyword.operator.word.in.js
+      pop: 1
+    - match: of{{identifier_break}}
+      scope: keyword.operator.word.of.js
```

Comment:
When changing scopes, I'd suggest to follow general scoping used for such operator keywords in other languages, such as [Python](https://github.com/sublimehq/Packages/blob/6a88bab4a59e6c5127bb89348232426bf135cf0c/Python/Python.sublime-syntax#L2288-L2289) instead of inventing something new again.

With a look onto the context name, it however might mean a bit more work as we now distinguish e.g. `in` as operator in conditional expressions and `in` keyword in for-loops.

`in` targets objects, `of` targets type checks.